pkg/model/openstackmodel/servergroup.go

```diff
@@ -183,23 +186,12 @@ func (b *ServerGroupModelBuilder) buildInstances(c *fi.ModelBuilderContext, sg *
 					c.AddTask(t)
 				}
 			}
-		} else if b.Cluster.Spec.CloudConfig.Openstack != nil && b.Cluster.Spec.CloudConfig.Openstack.Router == nil {
-			// No external router, but we need to add master fixed ips to certificates
-			if ig.Spec.Role == kops.InstanceGroupRoleMaster {
-				b.associateFixedIPToKeypair(instanceTask)
-			}
 		}
 	}
 
 	return nil
 }
 
-func (b *ServerGroupModelBuilder) associateFixedIPToKeypair(fipTask *openstacktasks.Instance) {
-	// Ensure the floating IP is included in the TLS certificate,
-	// if we're not going to use an alias for it
-	fipTask.ForAPIServer = true
-}
-
 func (b *ServerGroupModelBuilder) associateFIPToKeypair(fipTask *openstacktasks.FloatingIP) {
```

Comment:
I suspect you'll want to remove this as well, but that can be a separate PR.

Reply:
Yep. Either in #9554 or a follow-up.